src/command_line/main.cc

```diff
@@ -94,14 +111,14 @@ int main(int argc, char** argv) {
     node::index_tree(ti1, source_tree, ld, ucm);
     node::index_tree(ti2, destination_tree, ld, ucm);
     lgm_algorithm.init(ti2);
-    std::cout << lgm_algorithm.ted_k(ti1, ti2, k);
-    getrusage(RUSAGE_SELF, &after_rusage);
-    std::cout << " " << lgm_algorithm.get_subproblem_count();
-    timersub(&after_rusage.ru_utime, &before_rusage.ru_utime, &runtime_utime);
-    timersub(&after_rusage.ru_stime, &before_rusage.ru_stime, &runtime_stime);
-    runtime = runtime_utime.tv_usec + runtime_utime.tv_sec * 1000000 +
-        runtime_stime.tv_usec + runtime_stime.tv_sec * 1000000;
-    std::cout << " " << runtime << std::endl;
+    std::cout << "Distance TED_K:" << lgm_algorithm.ted_k(ti1, ti2, k) << std::endl;
+    std::cout << "Distance TED:" << lgm_algorithm.ted(ti1, ti2) << std::endl;
```

Comment:
LGM is not a standard TED algorithm. Moreover, this CLI wasn't meant for production :wink: 
As a quick fix, we could use APTED for `ted` and TouzetKRSetTreeIndex for `tedk`, if both types should be enabled. Instead of `lgm` argument value, I propose `ted` and `tedk` to choose one of the two algorithms. They should also not be executed together.

Reply:
I will do APTED only for now, as the improved CLI should let the user choose whichever algorithm available